docs/0001-digest-algorithms.md

```diff
@@ -20,4 +20,7 @@ This extension is an index of additional digest algorithms. It provides a contro
 
 ## Maintenance
 
-In order to have an additional digest algorithm listed here, please submit a pull request on this extension that adds it to the table. New entries should have a name that does not conflict with those defined in the [OCFL Specification](https://ocfl.io/latest/spec/) or this community extension, and is preferably in common use for the given algorithm. In the case that long description is required it may be appropriate to submit a new extension describing the algorithm along with an update to this extension that links to it.
+In order to have an additional digest algorithm listed here, please submit a pull request on this extension that:
+* Adds the algorithm to the table. New entries should have a name that does not conflict with those defined in the [OCFL Specification](https://ocfl.io/latest/spec/) or this community extension, and is preferably in common use for the given algorithm.
+* Creates a new version of this extension with the next available extension number, obsoleting the current one 
```

Comment:
I assume that this extension is no longer going to be parameterized, correct? So, there's effectively no way to reference it. Does it matter if a new version of this extension was created opposed to just updating it? If a new version was created, how would anyone know which version an object was using?

Reply:
I believe it will be parameterized but the PR hasn't been merged yet. Editors discussed versioning and decided to hold with RFC behaviour and create new extension numbers for additions. A new version doesn't cause the old one to vanish. Whatever version an object references will continue to be valid.

Reply:
Update post 20200602 Editorial discussion. We're not parameterizing but the versioning description remains to be consistent with how extensions are handled in in general.